### PR TITLE
chore: add embedded type form components

### PR DESCRIPTION
**Pull request by juniormp**

### CODE REVIEW

File: src/components/pages/FormPage/FormPage.tsx [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+        id={id!}
+        style={{
+          width: "100%",
+          minHeight: "100vh",
+        }}
+        hidden={{
+          patchwallet: user.patchwallet,
+          response_path: user.responsePath,
+          user_handle: user.userHandle,
+          user_name: user.userName,
```
Comment (milukove) [inline src/components/pages/FormPage/FormPage.tsx:45]: @juniormp Let's add user id as well?

File: src/components/pages/FormPage/FormPage.tsx [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+  belongsToWorkspace: boolean;
+};
+
+const FormPage = () => {
+  const { user } = useAppSelector(selectAppStore);
+  const { id } = useParams();
+  const [isWorkspaceValid, setIsWorkspaceValid] = useState(false);
+
+  useEffect(() => {
+    axios
```
Comment (milukove) [inline src/components/pages/FormPage/FormPage.tsx:19]: Let's extract this api call to a separate file in `/src/services` folder? See other files there for examples.

File: src/components/pages/FormPage/FormPage.tsx [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+type UsersSearchResponse = {
+  belongsToWorkspace: boolean;
+};
+
+const FormPage = () => {
+  const { user } = useAppSelector(selectAppStore);
+  const { id } = useParams();
+  const [isWorkspaceValid, setIsWorkspaceValid] = useState(false);
+
+  useEffect(() => {
```
Comment (milukove) [inline src/components/pages/FormPage/FormPage.tsx:18]: When doing async requests inside `useEffect`, I prefer to add a controller, to handle abortion when component unmounts. For example:
* https://github.com/grindery-io/grindery-wallet/blob/main/src/components/pages/ActivityPage.tsx#L36
* https://github.com/grindery-io/grindery-wallet/blob/main/src/components/pages/ActivityPage.tsx#L56-L58

This makes sure there are no memory leaks or unhandled responses.

**Review by milukove — CHANGES_REQUESTED** [review]

File: src/components/pages/FormPage/FormPage.tsx (hunk 1 above)
Comment (juniormp) [reply src/components/pages/FormPage/FormPage.tsx:45]: Sure, btw the paras are in underscore case because its the only patter Typeform allows.

**Review by milukove — APPROVED** [review]

**Pull request merged**